src/replit/ai/modelfarm/structs.py

```diff
@@ -86,3 +86,68 @@ class EmbeddingModelRequest(BaseModel):
 class EmbeddingModelResponse(BaseModel):
     metadata: Optional[EmbeddingMetadata] = None
     embeddings: List[Embedding]
+
+
+class Txt2ImgModelImageResult(BaseModel):
+    url: str
+    content_type: str
+    file_name: str
+    file_size: int
+    width: int
+    height: int
+
+
+class Txt2ImgModelResponse(BaseModel):
+    images: List[Txt2ImgModelImageResult]
+    seed: int
+
+    @classmethod
```

Comment:
How does other models handle this nested deserialization problem? E.g. `EmbeddingModelResponse` can contain a list of `Embedding`s but they don't define this method to deserialize.